mopidy_ytmusic/backend.py

```diff
@@ -69,7 +70,12 @@ def __init__(self, config, audio):
         if self.auth and not self.oauth:
             self.api = YTMusic(auth=self._ytmusicapi_auth_json)
         elif self.oauth:
-            self.api = YTMusic(auth=self._ytmusicapi_oauth_json)
+            oauth_credentials=OAuthCredentials(
+                client_id=config["ytmusic"]["oauth_client_id"],
+                client_secret=config["ytmusic"]["oauth_client_secret"],
+            )
+
+            self.api = YTMusic(auth=self._ytmusicapi_oauth_json, oauth_credentials=oauth_credentials)
```

Comment:
apparently `YTMusic` now only works with providing OAuth credentials - otherwise it goes into `UNAUTHORIZED` mode, making it not include the authorization token in `(self.api.)headers`